scripts/publicNames.py

```diff
@@ -6,7 +6,7 @@
 
 Tree = lambda: collections.defaultdict(Tree)
 
-EXCLUDE_DIRS = ['test', 'tests', 'plugin_tests', '.*egg-info']
+EXCLUDE_DIRS = ['test', 'tests', 'plugin_tests', '.*egg-info', 'web_client']
```

Comment:
Rather than blacklisting, can we just not add any symbol that isn't part of a python package? That way things like `mail_templates` or whatever other package_data doesn't get caught up in this list.